lib/manager/nuget/util.ts

```diff
@@ -56,12 +65,7 @@ export async function determineRegistries(
     return undefined;
   }
 
-  const registries = datasourceNuget.defaultRegistryUrls.map(
```

Comment:
Moved to separate function so that I could re-use it when filling the temporary config file.